doc/user/content/transform-data/patterns/durable-subscriptions.md

```diff
@@ -80,46 +172,28 @@ object will lead to increased resource utilization in Materialize.
     for that cluster to go up, which might require you to size up your cluster
     and consume additional compute credits.
 
-### Clean-up
+#### Best practices
 
-The history retention period represents the minimum amount of historical data
-guaranteed to be retained by Materialize. History clean-up is processed in the
-background, so older history may be accessible for the period of time between
-when it falls outside the retention period and when it is cleaned up.
-
-### Best practices
+- Given the additional memory costs associated with increasing the history
+  retention period on indexes, if you don't need the performance benefits of an
+  index, consider creating a materialized view for your subscription query and
+  configure the history retention period on that materialized view instead.
 
-Given the additional memory costs associated with increasing the history
-retention period on indexes, if you don't need the performance benefits of an
-index—in particular for the use case of durable subscriptions-you should
-consider creating a materialized view for your subscription query and configure
-the history retention period on that materialized view instead.
+- Similarly, because of the increased storage costs and processing time for the
+  additional historical data on whichever object has an increased history
+  retention period, you should consider configuring history retention period on
+  the index or materialized view directly powering the subscription, rather than
+  all the way through the dependency chain from the source to the index or
+  materialized view.
```

Comment:
```suggestion
- Similarly, because of the increased storage costs and processing time for the
  additional historical data on whichever object has an increased history
  retention period, consider configuring history retention period on the index
  or materialized view directly powering the subscription, rather than all the
  way through the dependency chain from the source to the index or materialized
  view.
```

Reply:
Nit, but "consider" vs. "you should consider" from your modification above.